src/renderers/common/Renderer.js

```diff
@@ -1123,12 +1132,19 @@ class Renderer {
 
 		let frameBufferTarget = this._frameBufferTarget;
 
-		if ( frameBufferTarget === null ) {
+		if ( frameBufferTarget === null || this._frameBufferTargetKey !== currentToneMapping ) {
+
+			if ( frameBufferTarget !== null ) frameBufferTarget.dispose();
+
+			// to improve performance we only want to use HalfFloatType if necessary
+			// HalfFloatType is required when a) tone mapping is configured or b) the backend specifically requests it
+
+			const type = ( currentToneMapping !== NoToneMapping || this.backend.needsHalfFloatFrameBufferTarget() ) ? HalfFloatType : UnsignedByteType;
```

Comment:
Side note: When using `UnsignedByteType`, it is in general recommended to use a sRGB storage format for the framebuffer. However, these formats are buggy in WebGL and do not consistently work across all platforms. Hence, sticking to `LinearSRGBColorSpace` provides more correct results.

See: https://stackoverflow.com/a/47098507/5250847